### ISSUE TITLE
Contao Manager 1.4.3: Opcode-Cache "Cache leeren" ohne Auswirkung

### ISSUE THREAD

**Issue by derMatze82** [contao/contao-manager#648]
Nach Klick auf den Button und dem erfolgreichen Absenden des Requests
`/contao-manager.phar.php/api/server/opcache?opcache_reset=#token#`
(Response `{"opcache_enabled":"1","reset_token":"a12345678909"}`)
wird der Opcode-Cache nicht geleert. Ich habe es über das [cachetool](https://github.com/gordalina/cachetool) verifiziert.
Es gibt auch kein Feedback im Contao-Manager selbst, ob das Leeren erfolgreich war oder nicht.
Beim cachetool musst ich ja noch das Socket mit übergeben (z.B. `--fcgi=/var/run/php/php8.0-fpm.sock`), muss das in Contao/Manager auch noch irgendwo konfiguriert werden?
System: Ubuntu 20, Apache 2.4, PHP 8.0.3, Contao 4.10.7

**Comment by aschempp** [comment]
kann es sein dass du mehrere FPM-Worker hast? Der Manager löscht den OpCache nur in seinem aktuellen Prozess (bzw. führt einfach `opcache_reset()` aus.

**Comment by derMatze82** [comment]
Nein, es gibt nur einen Pool. Wenn ich im Contao /web Verzeichnis eine PHP Datei erstelle, die `opcache_reset()` ausführt, und ausführe, sehe ich danach im cachetool bei "Last restart time" eine Änderung. Führe ich es über den Contao Manager aus, sehe ich keine Änderung im Cachetool
